crates/astria-proto/src/native/sequencer/v1alpha1.rs

```diff
@@ -209,6 +213,9 @@ impl SignedTransaction {
 pub struct UnsignedTransaction {
     pub nonce: u32,
     pub actions: Vec<Action>,
+    /// optional asset to use for fee payment.
+    /// if None, then the native asset is used.
+    pub fee_asset: Option<asset::Id>,
```

Comment:
Noted above, I would prefer if we made this behavior explicit rather than special-casing our "native" assets vs. non-native assets.